Bootstrap/Setup.php

```diff
@@ -86,6 +86,10 @@ public function update($oldVersion)
             $this->addIndexToPriceTable();
         }
 
+        if (version_compare($oldVersion, '2.1.1', '<')) {
```

Comment:
please use `version_compare($oldVersion, '2.1.1', '<=')` because the next version will be 2.1.2